osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -58,6 +61,11 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
             BeatmapSetInfo beatmapSetInfo = beatmap.BeatmapSetInfo;
             BeatmapInfo beatmapInfo = beatmap.BeatmapInfo;
 
+            string bpm = GetBPMRange(beatmap);
+            string length = "" + TimeSpan.FromMilliseconds((beatmap.Beatmap.HitObjects.Last().EndTime - beatmap.Beatmap.HitObjects.First().StartTime)).ToString(@"m\:ss");
+            string hitCircles = "" + beatmap.Beatmap.HitObjects.Count(b => b.GetType().ToString().Equals("osu.Game.Modes.Osu.Objects.HitCircle"));

```

Comment:
Why are you naming hitobjects `b`? `h` would fit way better. Also use `h is HitCircle` instead of this convoluted string comparison. Same for the other places in the code which are similar.